src/common/NoResults/styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import { ReactComponent as Picture } from "../../images/Picture.svg"
+
+export const StyledHeader = styled.div`
+    margin: 56px;
+`;
+
+export const StyledImages = styled(Picture)`
+    display: flex;
+    justify-content: center;
+    position: relative;
+    width: 100%;
+    height: 100%;
+`;
+
+export const NoResultsText = styled.p`
+text-align: center;
+margin-top: 20px;
+font-size: 18px;
+color: black;
```

Comment:
Ogólnie podmieniłbym to na `color: {({theme}) => theme.color.black};`, ale czy ta linijka jest w ogóle potrzebna? Czy tekst domyślnie nie jest czarny? 😄